src/DB.php

```diff
@@ -758,6 +766,7 @@ public static function insertMessageRequest(Message &$message)
             $sth->bindParam(':message_id', $message_id, \PDO::PARAM_INT);
             $sth->bindParam(':user_id', $from_id, \PDO::PARAM_INT);
             $sth->bindParam(':date', $date, \PDO::PARAM_STR);
+            $sth->bindParam(':edit_date', $edit_date, \PDO::PARAM_STR);
```

Comment:
This is so that the latest edit date is saved with the original message, right?
Unfortunately this doesn't get called when a message is edited, because `insertEditedMessageRequest()` is called, not `insertMessageRequest()`

Please correct me if I'm wrong!